kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/config/ReThinkSinkConfig.scala

```diff
@@ -18,7 +18,7 @@ package com.datamountaineer.streamreactor.connect.rethink.config
 
 import java.util
 
-import com.datamountaineer.streamreactor.temp.{ErrorPolicySettings, KcqlSettings, NumberRetriesSettings}
+import com.datamountaineer.streamreactor.temp._
```

Comment:
It's not something I'm against - if you're using implicit conversions, or most of the imports, then a wildcard import is more then OK - they shouldn't have an impact on performance...if it's only a couple (2,3...4...), then I typically prefer them separate, like they were before ({ErrorPolicySettings, KcqlSettings, NumberRetriesSettings})